src/common/snippets/include/snippets/lowered/linear_ir.hpp

```diff
@@ -9,33 +9,17 @@
 #include "snippets/lowered/expression.hpp"
 #include "snippets/target_machine.hpp"
 #include "snippets/shape_inference/shape_inference.hpp"
+#include "snippets/debug_caps.hpp"
```

Comment:
Minor (I see that `debug_caps.hpp` contains `ifdef` too but it's be better if the original files have these checks too)
```suggestion
#ifdef SNIPPETS_DEBUG_CAPS
#include "snippets/debug_caps.hpp"
#endif
```

Reply:
updated. thanks Alexandra!